### DISCUSSION TITLE
Equivalent constraints to V1 PrismaticJoint

### DISCUSSION THREAD

**Discussion by aboehmCT** [bepu/bepuphysics2#309] [Q&A]
Is there an equivalent joint to the V1 PrismaticJoint. I see that the V1 RevoluteJoint is replaced by hinge but I can't find an equivalent to the PrismaticJoint.

**Comment by RossNordby** [comment]
v1's PrismaticJoint was internally a `PointOnLineJoint` and a `NoRotationConstraint`. In v2, you could use a `PointOnLineServo` and `AngularServo`.

There's not currently a single constraint that solves all the degrees of freedom simultaneously, though there could be.
